src/api/useVehicleLocations.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * this is a custom hook that fetches the vehicle locations from the API.
+ * it recieves an interval of two dates, and loads locations of all vehicles in that interval.
+ * if some of the interval has already been loaded,
+ */
+
+import { useEffect, useState } from 'react'
+import { VehicleLocation } from 'src/model/vehicleLocation'
+
+const config = {
+  apiUrl: 'https://open-bus-stride-api.hasadna.org.il/siri_vehicle_locations/list?get_count=false',
```

Comment:
let's use the generated api? we don't want to maintain a couple of api access code

Reply:
can't find such thing